CONTRIBUTING.md

```diff
@@ -31,10 +32,35 @@ A maintainer will request changes if necessary. For content contributions, don't
 
 ## Setup Instructions
 
-This project is built using [Hugo](https://gohugo.io/), an open-source static site generator. To get the project up and running:
+This project is built using
+
+- [Hugo](https://gohugo.io/), an open-source static site generator
+- [Docker](https://www.docker.com/), a platform to generate containerized environments.
+
+Using Docker for the project is preferred to minimize conflicts arising from machine, OS, or dependency variations.
+
+### To get the project up and running Docker
```

Comment:
Semicolon after Docker, also maybe let's add "with Docker"

Reply:
Added a colon instead per [writing style guide](https://wmich.edu/writing/punctuation/colon#:~:text=The%20colon%20(%20%3A%20)%20and%20semicolon,should%20not%20be%20used%20interchangeably.)